web-starter/templates/lib/capistrano/tasks/drush.rake

```diff
@@ -5,31 +5,37 @@
 # - :sqlsync: Copies a database from a remote Drupal site, assumes ENV['source'] provided which is a drush alias
 # - :rsync: Copies files from a remote Drupal site, assumes ENV['source'] provided which is a drush alias
 # - :sqldump: Dumps the database to the current revision's file system
-# - :cc: Clears the entire Drupal cache
+# - :cr: (Drupal 8) Rebuilds the entire Drupal cache
+# - :cc: (Drupal 7) Clears the entire Drupal cache
 # - :update: Runs all pending updates, including DB updates, Features and Configuration -- if set to use those
 # - :updatedb: Runs update hooks
 # - :features:revert: Reverts Features, which may be all Features or just Features in particular directories
-# - :configuration:sync: Synchronizes Configuration and loads it from the Data Store to the Active Store
+# - :configuration:import: (Drupal 8) Import Configuration into the database from the config management directory
+# - :configuration:sync: (Drupal 7) Synchronizes Configuration and loads it from the Data Store to the Active Store
 # - :sapi:reindex: Clear Search API indexes and reindex each
 #
 # Variables:
 # - :drupal_features: Whether the Features module is enabled -- defaults to TRUE
-# - :drupal_cmi: Whether the Configuration module is enabled -- defaults to FALSE
 # - :drupal_features_path: Path(s) to scan for Features modules, if empty reverts all Features -- defaults to empty
+# - :drupal_cmi: (Drupal 7) Whether the Configuration module is enabled -- defaults to FALSE
 # - :drupal_db_updates: Whether to run update hooks on deployment -- defaults to TRUE
 
 namespace :load do
   task :defaults do
     set :drupal_features, true
-    set :drupal_cmi, false
     set :drupal_features_path, %w[]
+    set :drupal_cmi, false
     set :drupal_db_updates, true
     set :settings_file_perms, '644'
     set :site_directory_perms, '750'
   end
 end
 
 namespace :drush do
+  if fetch(:platform) == "drupal8"
```

Comment:
This only worked for me when set in the defaults, after setting directory permissions. Also the platform seems to be set in web.rake back to "drupal" and not what is set in deploy.rb